co2calculator/constants.py

```diff
@@ -138,6 +138,7 @@ class BusTrainRange(str, enum.Enum):
 
     LOCAL = "local"
     LONG_DISTANCE = "long-distance"
+    AVERAGE = "average"
```

Comment:
Average exists for trains but not for buses in the emission_factors_transport.csv file. So we need to split this enum into two classes: BusRange and TrainRange. Otherwise the test `test_compare_enums_with_data` in `test_data_code_compliance.py` fails

Reply:
Create issue #200